config/locales/forms.yml

```diff
@@ -115,6 +115,16 @@ en:
         salary: >-
           Enter the annual salary or salary range, and/or pay scale range.
           Where relevant, state if the salary is full-time equivalent
+      jobseekers_job_application_personal_details_form:
+        national_insurance_number: >-
+          It's on your National Insurance card, benefit letter, payslip or P60.
+          For example, 'QQ 12 34 56 C'.
+        phone_number: Please only give a phone number that you are happy to be contacted on.
+        previous_names: For example maiden name or name at birth, if different from the name you have now.
+      subscription_form:
```

Comment:
I'm not sure why the subscription_form and unsubscribe_form text is being added in this PR? (Also further down this file)

Reply:
I've removed them now. Not too sure how they got there either!